lib/Extension/Bundle/Traits/PackageVersionTrait.php

```diff
@@ -35,7 +35,7 @@ trait PackageVersionTrait
     abstract protected function getComposerPackageName(): string;
 
     /**
-     * {@inheritdoc}
+     * @return array|string|string[]|null
```

Comment:
how can that be an array? Shouldn't that be just a string?

Reply:
I think not even null should be allowed there...

Reply:
I saw that `preg_replace` has those return types that are possible. So, just used those. 
However, I realized that in this scenario, we might just be getting a string. So, changed.

Thanks.